packages/cspell-pipe/package.json

```diff
@@ -110,7 +135,17 @@
     "node": ">=14"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^24.0.0",
+    "@rollup/plugin-json": "^6.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "@rollup/plugin-typescript": "^11.0.0",
     "@types/node": "^18.11.18",
-    "jest": "^29.3.1"
+    "globby": "^13.1.3",
+    "jest": "^29.3.1",
+    "rollup": "^3.9.1",
+    "tslib": "^2.4.1"
+  },
+  "dependencies": {
+    "vitest": "^0.26.3"
```

Comment:
@Jason3S Is it intended to ship `vitest` as a dependency? I just saw it installed after upgrading `cspell`.